babel/src/main/java/org/apache/calcite/sql/babel/SqlBabelCreateTable.java

```diff
@@ -36,13 +58,21 @@ public class SqlBabelCreateTable extends SqlCreateTable {
   /** Creates a SqlBabelCreateTable. */
   public SqlBabelCreateTable(SqlParserPos pos, boolean replace,
       TableCollectionType tableCollectionType, boolean volatile_,
-      boolean ifNotExists, SqlIdentifier name, SqlNodeList columnList,
-      SqlNode query) {
-    super(pos, replace, ifNotExists, name, columnList, query);
+      boolean ifNotExists, SqlIdentifier name, @Nullable SqlNodeList columnList,
+      @Nullable SqlNode query) {
+    super(OPERATOR, pos, replace, ifNotExists, name, columnList, query);
     this.tableCollectionType = tableCollectionType;
     this.volatile_ = volatile_;
   }
 
+  @SuppressWarnings("nullness")
+  @Override public List<SqlNode> getOperandList() {
+    return ImmutableNullableList.of(SqlLiteral.createBoolean(getReplace(), pos),
+        SqlLiteral.createSymbol(tableCollectionType, pos),
```

Comment:
I’m taking the most straightforward approach using SqlLiteral for boolean flag for `replace` and i`fNotExists`. Ideally, `replace` and `ifNotExists` should each have their own `SqlNode` implementation.
That way, we won’t need to write code like this every time:
```
if (ifNotExists) {
  writer.keyword("IF NOT EXISTS");
}
```

Instead, we could just call:
```
ifNotExists.unparse(...)
```